src/basic/missing_syscall.h

```diff
@@ -53,32 +53,38 @@ static inline int missing_pivot_root(const char *new_root, const char *put_old)
 
 /* ======================================================================= */
 
-#if defined __x86_64__
-#  define systemd_NR_memfd_create systemd_SC_arch_bias(319)
-#elif defined __arm__
-#  define systemd_NR_memfd_create 385
-#elif defined __aarch64__
+#if defined(__aarch64__)
 #  define systemd_NR_memfd_create 279
-#elif defined __alpha__
+#elif defined(__alpha__)
 #  define systemd_NR_memfd_create 512
-#elif defined(__powerpc__)
-#  define systemd_NR_memfd_create 360
-#elif defined __s390__
-#  define systemd_NR_memfd_create 350
-#elif defined _MIPS_SIM
+#elif defined(__arc__) || defined(__tilegx__)
```

Comment:
are these archs actually related? or is this coincidence?

Reply:
All syscall numbers for these architectures are equivalent, and they have does not have specific numbers. The syscall numbers defined in include/uapi/asm-generic/unistd.h are used for those archs.